backend/sky_lectures/urls.py

```diff
@@ -13,9 +13,30 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
+from django.conf.urls import url
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
+from rest_framework import permissions, routers, serializers, viewsets
+from drf_yasg.views import get_schema_view
+from drf_yasg import openapi
+from django.contrib.auth.models import User
+
+schema_view = get_schema_view(
+   openapi.Info(
+      title="Snippets API",
+      default_version='v1',
+      description="Test description",
+      terms_of_service="https://www.google.com/policies/terms/",
+      contact=openapi.Contact(email="contact@snippets.local"),
+      license=openapi.License(name="BSD License"),
+   ),
+   public=True,
+   permission_classes=[permissions.AllowAny],
+)
```

Comment:
I guess there is no need to add permission_classes here due to the fact that swagger should be only active during the development.